keps/sig-auth/3331-structured-authentication-configuration/README.md

```diff
@@ -733,8 +734,6 @@ providers such as Okta, Azure AD, etc:
 #### Deprecation
 
 kube-apiserver `--oidc-*` flags require deprecation warnings on the stable release of the feature.
-It is possible to react only to the `--oidc-issuer-url` flag because other flags cannot be enabled separately from this one.
```

Comment:
We already check existence of any `oidc-*` flags as part of the initial implementation, so we can use the same post deprecation.